### PR TITLE
project: add template

### PR DESCRIPTION
**Pull request by rissson**

### CODE REVIEW

File: template/flake.nix [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+          };
+
+          packages = self.lib.overlayToPkgs self.overlays osPkgs;
+        }
+      );
+
+      outputs = {
+        lib = import ./lib { inherit lib; };
+
+        overlay = import ./pkgs;
```
Comment (rissson) [inline template/flake.nix:61]: Let's add an example for that.

File: template/flake.nix [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+        overlays = { };
```
Comment (rissson) [inline template/flake.nix:63]: Let's add an example for that.

File: template/flake.nix [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+        nixosModules = recursiveUpdate (import ./modules) {
+          profiles = import ./profiles;
```
Comment (rissson) [inline template/flake.nix:66]: Let's add an example workstation profile.

File: template/hosts/example/configuration.nix [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# NixOS configuration for host `example`
```
Comment (rissson) [inline template/hosts/example/configuration.nix:1]: Let's make a host that works with `build-vm`

File: template/lib/default.nix [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{ lib }:
+
+{
+  overlayToPkgs = import ./overlays-to-pkgs.nix { inherit lib; };
```
Comment (rissson) [inline template/lib/default.nix:4]: Maybe move this function to soxin.

Comment (rissson) [reply template/lib/default.nix:4]: And acknowledge it to https://github.com/nrdxp/nixflk. Let's also add an "acknowledgments" section in the readme.

**Comment by rissson** [comment]
@kalbasit this seems ready

**Review by kalbasit — APPROVED** [review]

**Pull request merged**